### ISSUE TITLE
KeyError: u'additional_folders'

### ISSUE THREAD

**Issue by edro15** [seunomosowon/TA-mailclient#20]
**OS details**
```bash
  Operating System: CentOS Linux 7 (Core)
  Kernel: Linux 3.10.0-1127.13.1.el7.x86_64
  Architecture: x86-64
```
**Steps performed**
* installed the app in a single server Splunk 7.3 instance, 
* configured `local/inputs.conf` following documentation, and 
* restarted splunk.

By troubleshooting using Splunk UI and the suggested command (`index=_internal sourcetype=splunkd (component=ModularInputs OR component=ExecProcessor) mail.py`), I verify that the following error has been thrown:
```bash
12-28-2020 10:58:04.377 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py" Top level exception:  u'additional_folders'
12-28-2020 10:58:04.377 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py" Traceback (most recent call last):
12-28-2020 10:58:04.377 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py"   File "/opt/splunk/etc/apps/TA-mailclient/bin/mail.py", line 382, in stream_events
12-28-2020 10:58:04.378 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py"     self._stream_events(inputs, ew)
12-28-2020 10:58:04.378 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py"   File "/opt/splunk/etc/apps/TA-mailclient/bin/mail.py", line 408, in _stream_events
12-28-2020 10:58:04.378 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py"     self.additional_folders = input_item['additional_folders'].split(',')
12-28-2020 10:58:04.378 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-mailclient/bin/mail.py" KeyError: u'additional_folders'
```

**Comment by seunomosowon** [comment]
I have tested a fix for this, It's currently being tested for all v7.x, and v8.x. I'll upload it to splunkbase in a few minutes.

**Comment by edro15** [comment]
Oh, I saw it while pushing the PR. :) 
I added a default value as well, to avoid other possible errors. 
Thanks for your work.

**Comment by edro15** [comment]
@seunomosowon thanks. Fixed in PR #23
